src/temp-humd.ino

```diff
@@ -0,0 +1,26 @@
+#include "ObserverSerial.h"
+#include "SensorTemp.h"
+#include "ThreadController.h"
+
+SensorTemp sensor;
```

Comment:
I would be more specific at name `SensorTemp` class, as there are different types of sensors.